### PR TITLE
Add method for pagination

### PR DESCRIPTION
**Pull request by alavieille**

### CODE REVIEW

File: WorkflowFunctionBundle/Tests/Functional/Repository/WorkflowFunctionRepositoryTest.php [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+    public function testCountFilterSearch($descriptionEntity, $columns, $search, $count)
+    {
+        $worflowFunctions = $this->repository->countFilterSearch($descriptionEntity, $columns, $search);
+        $this->assertEquals($count, $worflowFunctions);
+    }
+
+    /**
+     * @return array
+     */
+    public function provideColumnsAndSearchAndCount(){
```
Comment (nicolasThal) [inline WorkflowFunctionBundle/Tests/Functional/Repository/WorkflowFunctionRepositoryTest.php]: psr

File: WorkflowFunctionBundle/Tests/Functional/Repository/WorkflowFunctionRepositoryTest.php [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+    }
+
+    /**
+     * Generate columns of content with search value
+     *
+     * @param string $searchName
+     *
+     * @return array
+     */
+    protected function generateColumnsProvider($searchName = '')
```
Comment (nicolasThal) [inline WorkflowFunctionBundle/Tests/Functional/Repository/WorkflowFunctionRepositoryTest.php:108]: same remark on the position as in the user bundle

Comment (alavieille) [inline WorkflowFunctionBundle/Tests/Functional/Repository/WorkflowFunctionRepositoryTest.php:108]: this method is used by several method, currently is just under the last provider wich use it.

Comment (nicolasThal) [inline WorkflowFunctionBundle/Tests/Functional/Repository/WorkflowFunctionRepositoryTest.php:108]: all right

**Comment by alavieille** [comment]
@nicolasThal

**Comment by nicolasThal** [comment]
:+1:

**Pull request merged**